core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.dora;
+
+import static org.junit.Assert.assertTrue;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.cache.CacheManager;
+import alluxio.client.file.cache.CacheManagerOptions;
+import alluxio.client.file.cache.PageId;
+import alluxio.client.file.cache.PageMetaStore;
+import alluxio.conf.Configuration;
+import alluxio.conf.PropertyKey;
+import alluxio.grpc.File;
+import alluxio.grpc.FileFailure;
+import alluxio.util.io.BufferUtils;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class PagedDoraWorkerTest {
+  private PagedDoraWorker mWorker;
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+  private CacheManager mCacheManager;
+  private final long mPageSize =
+      Configuration.global().getBytes(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE);
+
+  @Before
+  public void before() throws Exception {
+    CacheManagerOptions cacheManagerOptions =
+        CacheManagerOptions.createForWorker(Configuration.global());
+
+    PageMetaStore pageMetaStore =
+        PageMetaStore.create(CacheManagerOptions.createForWorker(Configuration.global()));
+    mCacheManager =
+        CacheManager.Factory.create(Configuration.global(), cacheManagerOptions, pageMetaStore);
+    mWorker =
+        new PagedDoraWorker(new AtomicReference<>(1L), Configuration.global(), mCacheManager);
```

Comment:
I think we have dependency injection ready for this? @JiamingMai what's the recommended way of using dependency injection in unit tests?